### PR TITLE
add blog mauicestclair.fr

### PR DESCRIPTION
**Pull request by Kapusch**
:pencil2: **Blog Url**: https://www.mauicestclair.fr

:scroll: **Feed Url**: https://www.mauicestclair.fr/index.xml

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [ ] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by Kapusch** [comment]
Dear reviewers, please note that I have not ticked the '3 posts in last 6 months' guideline because I have published 3 of my first posts on the same day. 

Indeed, main reason is it could provide frustration to my first readers if I only published one. 
However, I have more than 10 articles ready to be published in my backlog, but will only release them on a regularly basis.

Hope you will understand and accept my blog, I really need more visibility ! 😊
Thanks in advance 🐵

**Comment by Kapusch** [comment]
Dear @jamesmontemagno (I'm such a great fan of what you do to the community !), as I'd like my French blog on .NET MAUI to be a bit more visible, could you please review and approve the PR whenever you have a bit of time ?
Thanks in advance for your help !

**Comment by Kapusch** [comment]
Hi @jfversluis , I have noticed you did merge some PRs in the past, whenever you have a bit of time (and I know it's been pretty pack with the awesome MediaElement !!), could you please review and approve it ?
Thanks in advance 😊

File: src/Firehose.WebCore/Properties/launchSettings.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
   "profiles": {
     "Firehose.WebCore": {
       "commandName": "Project",
-      "dotnetRunMessages": true,
```
Comment (jfversluis) [inline src/Firehose.WebCore/Properties/launchSettings.json:13]: Not sure if this should be changed?

**Comment by jfversluis** [comment]
Thanks for wanting to add your wonderful content to our little project @Kapusch!

I can definitely merge it, but it has been a while since I have been involved. So even if I do I think @jamesmontemagno might need to do a deploy before your feed shows up.

File: src/Firehose.WebCore/Properties/launchSettings.json (hunk 1 above)
Comment (Kapusch) [reply src/Firehose.WebCore/Properties/launchSettings.json]: Indeed, VS for Mac removes this line anytime I'm building the project.
I will fix it in my next commit.

**Comment by Kapusch** [comment]
Thank you very much Gerald !

**Review by jamesmontemagno — APPROVED** [review]

**Comment by Kapusch** [comment]
Thank you James !

**Pull request merged**